apps/typegpu-docs/src/content/examples/algorithms/mnist-inference/index.ts

```diff
@@ -455,12 +513,25 @@ export const controls = {
   Reset: {
     onButtonClick: resetDrawing,
   },
+  'Use Subgroups': {
```

Comment:
How about we only display the toggle if subgropus are supported?
```suggestion
  'Use Subgroups': hasSubgroups && {
```